src/ledger/transaction.ts

```diff
@@ -79,14 +80,16 @@ export const addTransactionOnUsPayment = async ({
   payerUser,
   payeeUser,
   memoPayer,
-}) => {
+  shareMemoWithPayee,
```

Comment:
why is this needed?

isn't that if there is a memo then this could be shared, and if there is not then this will be null?

Reply:
The function is called with `shareMemoWithPayee: true` only if it's a push payment.
From #301 ->
> Currently when a payment is being sent with payToUsername, the memo is not shared from the payer to the payee.